packages/tslint-config/tslint.json

```diff
@@ -25,6 +25,7 @@
         "curly": true,
         "custom-no-magic-numbers": [true, 0, 1, 2, 3, -1],
         "encoding": true,
+        "enum-snake-case": false,
```

Comment:
Did you mean to set this to `true` to enable the linter or are you planning to do that in a separate PR?

Reply:
I meant to leave it as false for this commit. I initially had it as true and started renaming everything, then had second thoughts because it seemed like it could be a big breaking case for small practical benefit. Do you think I should just go ahead and rename everything?

Reply:
I will do that in a separate PR. Maybe package by package.

Reply:
I would just do it. We need to bite the bullet and I don't expect there to be a TON on places where we haven't been following this rule, right?